### DISCUSSION TITLE
Github copilot pilot suddenly stopped working and I'm unable to login

### DISCUSSION THREAD

**Discussion by ShacharHarshuv** [community/community#41654] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I've been using Github copilot with Webstorm (and intellij IDE) for a while now, and I lately noticed it disconnected from my account. When I tried to log in again, though the browser UI said I authorized successfully, the IDE plugin still showed as not authenticated.

I tried reinstalling / updating both the IDE and the plugin, and I'm still unable to overcome this issue. 

Any idea what else I can try? Is this a new bug?

**Comment by queenofcorgis** [comment]
Hi @ShacharHarshuv,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have removed the duplicate discussions, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Reply by ShacharHarshuv** [reply]
Hey, I apologize for posting in the wrong place. Can you please let me know where can I get support for this issue?

**Comment by bchellingworth** [comment]
I get a similar issue, I have to restart my IDE, then go to tools->github copilot, it will say you are logged in, but just click logout. then restart IDE and go to tools->github copilot -> login. This usually works until the next time it randomly disconnects :)
